src/plot_api/plot_api.js

```diff
@@ -2171,7 +2169,9 @@ exports.update = function update(gd, traceUpdate, layoutUpdate, _traces) {
         if(relayoutFlags.layoutstyle) seq.push(subroutines.layoutStyles);
         if(relayoutFlags.axrange) {
             seq.push(
-                subroutines.doTicksRelayout,
+                function(gd) {
+                    return subroutines.doTicksRelayout(gd, relayoutSpecs.rangesAltered);
```

Comment:
N.B. @alexcjohnson I didn't implement per-axis "axrange" edit for `Plotly.react`. My main concern was improvement perf on dragend axis range relayout calls. I didn't adding an exception to the react diffing algo was worth it for this case.